middleware/locale/method/cookie_test.go

```diff
@@ -0,0 +1,33 @@
+package method_test
```

Comment:
Whether you place everything in a ```package locale``` or ```package locale_test``` depends on what you are doing: black-box or white-box testing.

If in doubt one uses black-box testing. (Go's usage needs more explaining than I am comfortable of giving here.)

When and if something breaks you want to triage it as fast as possible. With, allow me to call it that way, a »small interface and deep implementation« makes that hard: It's not fine-grained enough to single out a particular line, for example.

Traditionally *Caddy* has black-box tests. Even if you don't do these here, someone taking over your work expects that. So it's actually your usage that needs the justification.